doc/team.rst

```diff
@@ -0,0 +1,90 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+
+We are an international team dedicating to building a Pythonic API for GMT that makes GMT more accessible to
+new users, interfaces with the GMT C API without system calls, supports rich display in Jupyter notebooks,
+and integrates with the PyData ecosystem.
```

Comment:
Minor tweaks to the text and some word wrapping (not sure if we should wrap to 79 char or 88 char tbh, I usually do 79 char out of habit).

```suggestion
We are an international team dedicated to building a Pythonic API for the
Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
new users, by creating user-friendly interfaces with the GMT C API, supporting
rich display in Jupyter notebooks, and integrating with the PyData ecosystem.
```